buildchain/requirements-Linux.txt

```diff
@@ -4,13 +4,48 @@
 #
 #    tox -e pip-compile
 #
-cloudpickle==0.8.1 \
-    --hash=sha256:3ea6fd33b7521855a97819b3d645f92d51c8763d3ab5df35197cd8e96c19ba6f \
-    --hash=sha256:70b492d5e06b779707c00b334311abcd4ac144c56910068aeeda0079e6244ef0 \
+certifi==2019.3.9 \
+    --hash=sha256:59b7658e26ca9c7339e00f8f4636cdfe59d34fa37b9b04f6f9e9926b3cece1a5 \
```

Comment:
Why do we have hash for Linux but not for Darwin?

Reply:
Because I don't have a mac :D - @gdemonet introduced a `tox` target for deps which fixes hashes.

Reply:
We should ask a mac use to run it then, to keep both in sync.

Reply:
My bad, we actually merged this `pip-compile` target without updating the `requirements-Darwin.txt`